modules/ocf_irc/files/anope/chanserv.conf

```diff
@@ -0,0 +1,484 @@
+# Comments trimmed for easier editing.
```

Comment:
Well, I tried overridding some config later in the file that had been defined before and anope segfaulted, so I think sed would be the best option apart from including full config files like this.